docs/source/_static/install_dependencies.sh

```diff
@@ -112,7 +112,8 @@ elif [ -f /etc/os-release ]; then
     if [[ "$ID" == "ubuntu" || "$ID" == "debian" || "$ID_LIKE" == "ubuntu" || "$ID_LIKE" == "debian" || "$ID_LIKE" == "ubuntu debian" ]]; then
         if [[ ! $(uname -m) =~ ^arm* ]]; then
             sudo apt-get update
-            if [[ "$VERSION_ID" > "22.04" || "$VERSION_ID" == "22.04" ]]; then
+            UBUNTU_VERSION=$(inxi -Sx | grep -oP '(?<=Ubuntu\s)\d+\.\d+')
```

Comment:
inxi is not a default tool

consider something like
```suggestion
            UBUNTU_VERSION=$(cat /etc/lsb-release | awk -F '=' '/^DISTRIB_RELEASE=/ {{ print $2 }}')
```

Reply:
Hi @AnesHadzi, 
the /etc/lsb-release doesn't return the correct version on Ubuntu Mint (I get 21.1 codename Vera). The correct info can be found in /etc/upstream-release/lsb-release, but I have checked that stock Ubuntu doesn't have that directory. I'm looking for something that would work across all platofrms, but haven't found anything just yet.
Thanks, 
Jaka

Reply:
I do not have Mint to try, but IIRC ID from etc/os-release should say something like *mint*, and the system already rely on that param, you can do:
add if [Mint] cat /etc/upstream-release/lsb-release else cat /etc/lsb-release